{
    "type": "bundle",
    "id": "bundle--7a124596-68a7-4eea-a6c2-d87c505c9874",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--31118946-efaf-42fc-b0ca-27cfa1f8554c",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-04-02T00:16:20.161182Z",
            "modified": "2024-04-02T00:16:20.161182Z",
            "name": "CVE-2024-27212",
            "description": "In init_data of , there is a possible out of bounds write due to a missing bounds check. This could lead to local escalation of privilege with no additional execution privileges needed. User interaction is not needed for exploitation.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-27212"
                }
            ]
        }
    ]
}